content/blog/2024-05-20-summer-software-open-science-school.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Applications now open for a Summer School in Open Science + Research Software Engineering"
+date: 2024-05-20
+author: "Madicken Munk"
+---
+
+
+__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills would like to build on and expand them?__
+
+If this sounds like you, then you might be interested in the upcoming Summer School in Open Science and Research Software Engineering. 
+In July 2024, we will be hosting a five day workshop on open science and research software engineering at the University of Illinois at Urbana-Champaign. 
+
+This workshop complements previous summer and winter schools hosted by URSSI on research software engineering. 
+At this school, students will hone their open science skills in addition to building their skillset in research software engineering.  
+
+This is aimed at early-career researchers, particularly graduate students and postdocs, who are familiar with basic skills such as interacting with the Unix shell, version control using Git, and Python programming, and would like to learn more about best practices for developing research software and leveraging their research software to practice and enhance their own open science. 
+
+## Target Audience
+
+Ideal candidates for this workshop are science practitioners that user or develop software in their research and want to share their software in their community of practice, or are contributing to other research projects. 
+These practitioners want to make their research open, accessible, and reproducible by implementing open science best practices in addition to building and contributing to research software. 
+
+To get the most benefit from this workshop, we expect students to be familiar with the Unix shell, Python, and git, at the level taught at a Software Carpentry Workshop. 
+
+## Format and topics
+
+This five-day workshop will enable learners to hone their skills in developing sustainable research software, practicing open science in their workflow, and contributing to their communities of practice. Topcs covered will include:  
+
+*   The Ethos of Open Science
+*   Open Tools and Resources
+*   Open Data 
+*   Open Results 
+*   Software design and modularity
+*   Collaborative software development via Git+GitHub
+*   Software testing in Python
+*   Code review
+*   Packaging and distributing Python software
+*   Documentation
+*   Software, Data, and Documentation Licensing
+*   Reproducibility
+
+The school will consist of lectures on these topics along with open hacking time to allow participants to practice the concepts covered in the lectures. To facilitate the hands-on experience, each participant will bring a project to work on throughout the course for applying these concepts.
```

Comment:
```suggestion
The school will consist of lectures on these topics along with open hacking time to allow participants to practice the concepts covered in the lectures. To facilitate the hands-on experience, each participant should bring a project to work on throughout the course for applying these concepts.
```

Reply:
or "must"?

Reply:
What do you think about "is strongly encouraged to"?

Reply:
sure. Maybe also say this earlier, such as by line 16, to emphasize it